laterpay/asset_sources/scss/components/_debugger.scss

```diff
@@ -71,121 +77,105 @@
 
     .lp_debugger__close-link {
         color: $clickable;
-        font-weight: bold;
+        display: inline-block;
         height: $fs--2;
         line-height: 2;
-        margin: $fs--025 $fs--025 0 0;
         text-align: center;
-        transform: rotate(90deg);
+        transform: rotate(90deg) translate(2px, 3px);
         transition: transform 0.2s;
         width: $fs--2;
 
-        &:before {
-            line-height: 2;
-            padding: 0 0 0 ($fs * 0.1);
-            top: auto;
-        }
-
         &:hover {
             color: $clickable--highlight;
         }
-
-        .lp_debugger.lp_is-hidden & {
-            transform: rotate(-90deg);
-        }
     }
-}
 
 
-// debugger pane tabs --------------------------------------------------------------------------------------------------
-.lp_debugger-tabs {
-    border-bottom: 1px solid $border--darker;
-    margin: $fs--075 $fs--075 0;
-    position: relative;
-    z-index: $zindex;
-}
+    // debugger pane tabs --------------------------------------------------------------------------------------------------
+    .lp_debugger-tabs {
+        border-bottom: 1px solid $border--darker;
+        margin: $fs--075 $fs--075 0;
+        position: relative;
+        z-index: $zindex;
+    }
 
-.lp_debugger-tabs__link {
-    color: $clickable;
-    display: block;
-    padding: $fs--05 $fs;
+    .lp_debugger-tabs__link {
+        color: $clickable;
+        display: block;
+        padding: $fs--05 $fs;
 
-    &:hover {
-        color: $clickable--highlight;
+        &:hover {
+            color: $clickable--highlight;
 
-        span {
-            background-color: $clickable--highlight;
+            .lp_badge {
+                background-color: $clickable--highlight;
+            }
         }
-    }
-
-    span {
-        background: $clickable;
-        border-radius: $border-radius;
-        color: $text--inverse;
-        display: inline-block;
-        margin-left: $fs--05;
-        min-width: $fs--075;
-        padding: 1px 5px;
-        text-align: center;
-    }
-}
 
-.lp_debugger-tabs__item {
-    border: 1px solid transparent;
-    border-radius: $border-radius $border-radius 0 0;
-    float: left;
-    height: ($fs * 2.5);
-    list-style: none outside none;
-    margin: 0 2px -1px 0;
-    padding: 0;
-
-    &:hover {
-        border-color: $border--darker $border--darker darken($background, 5%);
+        .lp_badge {
+            background: $clickable;
+            margin-left: $fs--05;
+        }
     }
 
-    &.lp_is-selected {
-        border-color: $border--darker $border--darker darken($background, 5%);
+    .lp_debugger-tabs__item {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `lp_debugger-tabs__item` should be written in lowercase with hyphens